docs/Resources-Appendices/data-conversions.md

```diff
@@ -19,7 +21,7 @@ This chapter describes how to convert across binary, hexadecimal and decimal rep
 
 ## Binary - Hexadecimal
 
-The most convenient base for storing byte-wise information is hexadecimal (base 16). Two hexadecimal (base 16) digits can represent one byte of information. Each hexadecimal digit represents 4 bits of binary information.
+The most convenient base for storing byte-wise information is hexadecimal (base 16). Two hexadecimal digits can represent one byte of information. Each hexadecimal digit represents 4 bits of binary information.
```

Comment:
Why in `Two hexadecimal` `(base 16)`is removed?

Reply:
Oh I thought it wouldn't need to be repeated, I'll add it back.